contracts/finance/andromeda-set-amount-splitter/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "andromeda-set-amount-splitter"
-version = "1.1.0-beta"
+version = "2.0.0-b.1"
```

Comment:
This should be 1.2.0-b.1 right? I don't think adding CW20 support is a breaking change